ledspicer/data/ledspicer.conf

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LEDSpicer
+    logLevel="Info"
+    port="16161"
+    version="1.0"
+    type="Configuration"
+    fps="30"
+    userId="1000"
```

Comment:
you need to be sure that this IDs will be the correct user IDs or some plugins will fail.